sim/robots/flood_fill.py

```diff
@@ -513,10 +567,13 @@ def in_goal(*cells: tuple[int, int]) -> bool:
         else:
             info.reset_color_if(unknown_color)
             info.visited = 1
-    return groups, reduce(or_, groups.iter_sets(), set())
 
+    return groups, \
+        reduce(or_, groups.iter_sets(), set()), \
+        _mark_deadends_flood(maze, groups.iter_sets(), goals, deadend_color)
```

Comment:
Avoid `\` if possible:
```suggestion
    return (
        groups,
        reduce(or_, groups.iter_sets(), set()),
        _mark_deadends_flood(maze, groups.iter_sets(), goals, deadend_color),
    )
```